core/src/main/java/org/openedx/core/presentation/settings/VideoQualityFragment.kt

```diff
@@ -75,13 +80,19 @@ class VideoQualityFragment : Fragment() {
             OpenEdXTheme {
                 val windowSize = rememberWindowSize()
 
-                val videoQuality by viewModel.videoQuality.observeAsState(viewModel.currentVideoQuality)
+                val title = stringResource(
+                    id = if (viewModel.getQualityType() == VideoQualityType.Streaming)
+                        R.string.core_video_streaming_quality else
+                        R.string.core_video_download_quality
+                )
```

Comment:
Difficult to read construction.
Not required, but `else` could be moved to a separate line.

Reply:
done